test/grpc-creds/generate.sh

```diff
@@ -11,20 +11,50 @@ command -v minica >/dev/null 2>&1 || {
   exit 1;
 }
 
-for SERVICE in admin-revoker expiration-mailer ocsp-responder consul \
-  wfe akamai-purger bad-key-revoker crl-updater crl-storer \
-  health-checker; do
-  minica -domains "${SERVICE}.boulder"
+# On first-run, minica will generate a root keypair in the current directory.
+# We'll use that fact to run other keypair generations in parallel, otherwise
+# many minicas will race determining whose root gets used.
+# grpc/creds/creds.go:
+minica -domains "creds-test" -ip-addresses "127.0.0.1"
+
+SERVICE_WITH_SINGLE_SAN=(
+  "admin-revoker"
+  "expiration-mailer"
+  "ocsp-responder"
+  "consul"
+  "wfe"
+  "akamai-purger"
+  "bad-key-revoker"
+  "crl-updater"
+  "crl-storer"
+  "health-checker"
+)
+for SERVICE in ${SERVICE_WITH_SINGLE_SAN[@]}; do
+  minica -domains "${SERVICE}.boulder" &
 done
 
-for SERVICE in publisher nonce ra ca sa va rva ; do
-  minica -domains "${SERVICE}.boulder,${SERVICE}1.boulder,${SERVICE}2.boulder"
+SERVICE_WITH_MULTIPLE_SANS=(
+  "publisher"
+  "nonce"
+  "ra"
+  "ca"
+  "sa"
+  "va"
+  "rva"
+)
+for SERVICE in ${SERVICE_WITH_MULTIPLE_SANS[@]}; do
+  minica -domains "${SERVICE}.boulder,${SERVICE}1.boulder,${SERVICE}2.boulder" &
 done
 
-minica -ip-addresses 10.77.77.77,10.88.88.88
+minica -ip-addresses 10.77.77.77,10.88.88.88 &
 
-# grpc/creds/creds.go:
-minica -domains "creds-test" -ip-addresses "127.0.0.1"
+# These files are used by the TestTLSConfigLoad unit test
```

Comment:
I don't think we want any of the files generated by this script to be used by unit tests. (It's true that there are some used by unit tests today; I'm working on fixing that in https://github.com/letsencrypt/boulder/issues/7476.) 

Instead, the unit tests can either test functionality starting with already-in-memory certs, or they can test loading-from-disk functionality by writing files using [os.TempDir](https://pkg.go.dev/os#TempDir) and os.WriteFile.